graph/meta.go

```diff
@@ -23,13 +23,13 @@ const (
 	HasFaultInjection     MetadataKey = "hasFaultInjection"
 	HasHealthConfig       MetadataKey = "hasHealthConfig"
 	HasMirroring          MetadataKey = "hasMirroring"
-	HasMissingSC          MetadataKey = "hasMissingSC"
 	HasTCPTrafficShifting MetadataKey = "hasTCPTrafficShifting"
 	HasTrafficShifting    MetadataKey = "hasTrafficShifting"
 	HasRequestRouting     MetadataKey = "hasRequestRouting"
 	HasRequestTimeout     MetadataKey = "hasRequestTimeout"
 	HasVS                 MetadataKey = "hasVS"
 	HasWorkloadEntry      MetadataKey = "hasWorkloadEntry"
+	IsAmbient             MetadataKey = "isAmbient"
```

Comment:
If we remove from the Node then I guess we would not need this, either.

Reply:
right!